catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/controller/SearchControllerTest.java

```diff
@@ -258,6 +243,96 @@ public void testMetacardTypeValuesCacheEnabled() throws Exception {
         assertReplies(replies);
     }
 
+    @Test
+    public void testReducedMaxQueryResults() throws Exception {
+
+        framework = createFramework(20);
```

Comment:
Will leave comment, but I don't want to alter the setup when they are only used for a single test.